index.html

```diff
@@ -6,6 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Landing Page</title>
     <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Roboto">
```

Comment:
We might be getting all the Robotos, not just the medium non-italicised font. You can do it like this so that you don't get all the fonts. But this is just optional.

<img width="1512" alt="Screenshot 2022-11-29 at 9 56 22 AM" src="https://user-images.githubusercontent.com/2204029/204419540-4efb294e-e6d2-4d1b-8c09-5bbfa722aca7.png">